src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+
+# Some useful test classes inherited from avocado.Test
+
+from __future__ import print_function
+
+import os
+import json
+
+from avocado import Test as avocadoTest
+from avocado import skip
```

Comment:
Does it make sense to move to cancel() due to the warning in https://avocado-framework.readthedocs.io/en/52.0/WritingTests.html#test-skip-method and I assume its implementation will follow the definition of CANCEL ("test was canceled somewhere during the setUp(), the test method or the tearDown()") rather than SKIP ("test’s pre-requisites were not satisfied and the test’s body was not executed (nor its setUp() and tearDown)") - the skip* decorators are more in line with SKIP

Reply:
This is the skip _decorator_ that is being imported.  The `skip()` you are referring to is a method on the `Test` class which doesn't get imported individually but is in the `Test` class itself.